### ISSUE TITLE
Suggest for text change on landing page

### ISSUE THREAD

**Issue by rinva** [janelia-cellmap/openorganelle-blog#6]
@d-v-b I think it would look nice if the text at the bottom of the landing page was changed from "Welcome to HHMI Janelia's OpenOrganelle" to "About HHMI Janelia's OpenOrganelle".  I feel like a big "Welcome" would be better suited at the top of a page and it looks funny to me that it's at the bottom.

**Comment by d-v-b** [comment]
I agree! I think "about" is much better than "welcome to". I can get this done in the next few days.

**Comment by d-v-b** [comment]
@rinva this fix should be live now, let me know if you have any other issues!
